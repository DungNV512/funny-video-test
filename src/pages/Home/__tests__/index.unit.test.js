import React from 'react';
import { render } from '@testing-library/react';
import { HomeModule } from '..';

describe('HomeModule', () => {
  it('matches snapshot', () => {
    const { container } = render(<HomeModule />);
    // eslint-disable-next-line testing-library/no-container
    expect(container.cloneNode(true)).toMatchSnapshot();
  });
});
