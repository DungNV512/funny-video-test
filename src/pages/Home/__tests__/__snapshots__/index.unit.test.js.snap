// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeModule matches snapshot 1`] = `
<div>
  <div
    class="home-page"
  >
    <p
      class="label text-center"
    >
      Empty
    </p>
  </div>
</div>
`;
